modules/ROOT/pages/dataweave-cookbook-flat-file-read-and-write.adoc

```diff
@@ -0,0 +1,282 @@
+  <flow name="WriteFlatFile" >
+    <scheduler doc:name="Scheduler" >
+    	<scheduling-strategy >
+        <fixed-frequency frequency="10000" />
+    	</scheduling-strategy>
+    </scheduler>
+    <set-payload doc:name="Set Payload" doc:id="b2c2dc46-dd74-4d95-bde3-e6b761055468" value='#[{&#10;  "Batch": [&#10;    {&#10;      "TDR": [&#10;        {&#10;          "Record Type": "BAT",&#10;          "Sequence Number": 2,&#10;          "Amount": 32876,&#10;          "Account Number": "0123456789",&#10;          "Batch Function": "D",&#10;          "Type": "CR"&#10;        },&#10;        {&#10;          "Record Type": "BAT",&#10;          "Sequence Number": 3,&#10;          "Amount": 87326,&#10;          "Account Number": "0123456788",&#10;          "Batch Function": "D",&#10;          "Type": "CR"&#10;        }&#10;      ],&#10;      "BCF": {&#10;        "Record Type": "BAT",&#10;        "Sequence Number": 4,&#10;        "Batch Transaction Count": 2,&#10;        "Batch Transaction Amount": 120202,&#10;        "Unique Batch Identifier": "A000000001",&#10;        "Batch Function": "T",&#10;        "Type": "CR"&#10;      },&#10;      "BCH": {&#10;        "Record Type": "BAT",&#10;        "Company Name": "ACME RESEARCH",&#10;        "Sequence Number": 1,&#10;        "Unique Batch Identifier": "A000000001",&#10;        "Batch Function": "H"&#10;      }&#10;    },&#10;    {&#10;      "TDR": [&#10;        {&#10;          "Record Type": "BAT",&#10;          "Sequence Number": 6,&#10;          "Amount": 3582,&#10;          "Account Number": "1234567890",&#10;          "Batch Function": "D",&#10;          "Type": "DB"&#10;        },&#10;        {&#10;          "Record Type": "BAT",&#10;          "Sequence Number": 7,&#10;          "Amount": 256,&#10;          "Account Number": "1234567891",&#10;          "Batch Function": "D",&#10;          "Type": "CR"&#10;        }&#10;      ],&#10;      "BCF": {&#10;        "Record Type": "BAT",&#10;        "Sequence Number": 8,&#10;        "Batch Transaction Count": 2,&#10;        "Batch Transaction Amount": 3326,&#10;        "Unique Batch Identifier": "A000000002",&#10;        "Batch Function": "T",&#10;        "Type": "DB"&#10;      },&#10;      "BCH": {&#10;        "Record Type": "BAT",&#10;        "Company Name": "AJAX EXPLOSIVES",&#10;        "Sequence Number": 5,&#10;        "Unique Batch Identifier": "A000000002",&#10;        "Batch Function": "H"&#10;      }&#10;    }&#10;  ],&#10;  "RQF": {&#10;    "Record Type": "RQF",&#10;    "File Batch Count": 2,&#10;    "File Transaction Count": 8,&#10;    "File Transaction Amount": 116876,&#10;    "Unique File Identifier": "A000000001",&#10;    "Type": "CR"&#10;  },&#10;  "RQH": {&#10;    "Record Type": "RQH",&#10;    "File Creation Time": "10:10:00",&#10;    "File Creation Date": "2018-09-01",&#10;    "Currency": "USD",&#10;    "Unique File Identifier": "A000000001"&#10;  }&#10;}]'/>
```

Comment:
The set payload could be replaced with a File reader of the output of the "read flatfile example" right? Just to have this a little bit cleaner and understandable.